ast/Helpers.h

```diff
@@ -234,28 +234,26 @@ class MK {
     }
 
     static std::unique_ptr<MethodDef> Method(core::Loc loc, core::Loc declLoc, core::NameRef name,
```

Comment:
Can we make it explicit in the name?

Reply:
> Can we make it explicit in the name?

Happy to. Do you mind if i do this in a follow up PR? I have a change stacked on this branch that I don't want to have to fix the conflicts for.

Reply:
I'd actually prefer it done in this PR.
I want to see all affected callsites, as you're changing the default

Reply:
#2674